tanjun/abc.py

```diff
@@ -578,6 +735,42 @@ async def respond(
             typing.Union[hikari.SnowflakeishSequence[hikari.PartialRole], bool]
         ] = hikari.UNDEFINED,
     ) -> hikari.Message:
+        """Respond to this context.
+
+        Parameters
+        ----------
+        content : hikari.UndefinedOr[typing.Any]
+            The content to respond with.
+
+        Other Parameters
+        ----------------
+        ensure_result : bool
+            If provided and set to `True`, The response will be fetched performing an API call.
```

Comment:
On MessageContext ensure_result actually does nothing, it's just necessary to keep the signature compatible with Context and should likely be documented here as such